packages/rancher-istio/charts/templates/istio-uninstall-job.yaml

```diff
@@ -12,10 +12,12 @@ spec:
     spec:
       containers:
         - name: istioctl-uninstaller
-          image: rancher/istio-installer:{{ .Values.installerVersion }}
+          image: brendarearden/istio-test:4.1.6
```

Comment:
will update when new rancher/istio-installer is available prior to merging this pr